apps/allocations/app/components/Panel/NewAllocation.js

```diff
@@ -155,12 +166,14 @@ class NewAllocation extends React.Component {
       onChange: this.changePayoutToken,
     }
 
-    const warningMessages = (
-      <WarningMessage hasWarning={transferEnabled} type={'transferWarning'} />
+    const amountWarningMessages = (
+      <WarningMessage hasWarning={state.payoutTokenIndex !== 0} type={'tokenTransferWarning'} />
     )
 
-    const errorMessages = [ 'allocationError', 'addressError', 'descriptionError' ].map((e, i) => (
-      <ErrorMessage key={i} hasError={state[e]} type={e} />
+    const errorMessages = [ 'allocationError', 'addressError', 'descriptionError', 'ethBalanceError', 'tokenBalanceError' ].map((e, i) => (
+      <div>
+        <ErrorMessage key={i} hasError={state[e]} type={e} />
```

Comment:
It seems that the key prop need to go into the parent div, otherwise a react warning is showing up